### PR TITLE
Removing statistics tracking

### PR DESCRIPTION
**Pull request by EugeneKoryakin**
Signed-off-by: Your Real Name your.email@email.com

### CODE REVIEW

File: resources/META-INF/plugin-release-info.xml [hunk 1]
```diff
@@ -1,35 +1,28 @@
 <idea-plugin>
 
-    <id>ccom.intellij.idea.plugin.hybris.impex</id>
-    <name>hybris integration</name>
-    <version>10.5.1</version>
+    <id>com.intellij.idea.plugin.sap.commerce</id>
+    <name>SAP Commerce integration</name>
+    <version>2019.1</version>
```
Comment (MykytaKostiuk) [inline resources/META-INF/plugin-release-info.xml]: Please change version to 1908

File: resources/META-INF/plugin-release-info.xml [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
 
     <!-- http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html -->
     <idea-version since-build="191"/>
 
-    <vendor email="AlexanderBartash@gmail.com"
-            url="https://hybris-integration.atlassian.net/wiki/display/IIPPS/Intellij+IDEA+Plugin+Public+Space">
-        Alexander Bartash
+    <vendor email="EugeneKoryakin@gmail.com"
```
Comment (MykytaKostiuk) [inline resources/META-INF/plugin-release-info.xml]: Please change to hybrisideaplugin@epam.com

File: resources/META-INF/plugin-release-info.xml [hunk 3]
```diff
@@ -40,72 +33,21 @@
...
-              <li>Michael Golubev</li>
-              <li>Nicko Cadell</li>
-              <li>Dan Wanigasekera</li>
-              <li>Markus Priegl</li>
-              <li>Sergei Aksenenko</li>
-              <li>Roger Ye</li>
-              <li>Hector Longarte</li>
-              <li>Fabian Necci</li>
-              <li>Markus Perndorfer</li>
-              <li>FAIR Consulting Group</li>
```
Comment (MykytaKostiuk) [inline resources/META-INF/plugin-release-info.xml]: I think we can align contributors list with the README.md file

File: resources/META-INF/plugin-release-info.xml (hunk 1 above)
Comment (EugeneKoryakin) [reply resources/META-INF/plugin-release-info.xml]: @MykytaKostiuk commit reverted to include only Statistics-related changes.

File: resources/META-INF/plugin-release-info.xml (hunk 2 above)
Comment (EugeneKoryakin) [reply resources/META-INF/plugin-release-info.xml]: @MykytaKostiuk commit reverted to include only Statistics-related changes.

File: resources/META-INF/plugin-release-info.xml (hunk 3 above)
Comment (EugeneKoryakin) [reply resources/META-INF/plugin-release-info.xml]: @MykytaKostiuk commit reverted to include only Statistics-related changes.

**Pull request merged**